php/razorpay_js/app.js

```diff
@@ -0,0 +1,138 @@
+/**
+*
+* This is a basic node js server Created
+* for showing a demo on how to implement razorpay
+* in chargebee application
+* 
+*/
+require('dotenv').config();
+const express = require('express'),
+    cors = require('cors'),
+    qs = require('qs'),
+    axios = require('axios');
+
+const app = express()
+app.use(cors())
+app.use(express.json());
+app.use(express.urlencoded());
+
+
+/**
+ * Env Variables
+ */
+ const CB_AUTH_TOKEN = Buffer.from(process.env.CB_API_KEY + ":").toString('base64');
+ const CB_AUTH_HEADER = `Basic ${CB_AUTH_TOKEN}`;
+
+ const RP_AUTH_TOKEN = Buffer.from(process.env.RAZORPAY_KEY + ":" + process.env.RAZORPAY_KEY_SECRET).toString('base64');
+ const RP_AUTH_HEADER = `Basic ${RP_AUTH_TOKEN}`;
+
+/**
+ * [ @Chargebee ]
+ * Used to create chargebee's estimation on subscription for items
+ */
+app.post('/api/create_estimate', (req, res) => {
+    let data = qs.stringify(req.body);
+    let requestOptions = {
+        method: 'POST',
+        headers: {
+            "Authorization": CB_AUTH_HEADER,
+            "Content-Type": "application/x-www-form-urlencoded"
+        },
+        data: data,
+        url: `https://${process.env.SITE_ID}.chargebee.com/api/v2/customers/${process.env.CB_CUSTOMER_ID}/create_subscription_for_items_estimate`
+    };
+
+    axios(requestOptions)
+        .then(function (apiResult) {
+            res.status(apiResult.status).json(apiResult.data);
+        })
+        .catch(function (error) {
+            console.log(error.response.data);
+        });
+})
+
+/**
+ * [ @Razorpay ]
+ * Used to create customer on razorpay
```

Comment:
Used to create a customer in Razorpay.
Refer to this [API](http://API_LINK).

@cb-pravinrajmohan Kindly add the link for the respective API.